docs/apps.rst

```diff
@@ -81,3 +81,5 @@ Games
 .. automodule:: apps.play2048
 
 .. automodule:: apps.snake
+
+.. automodule:: apps.flappybird
```

Comment:
It's an optional app, so it should be `Flappybird` instead of `apps.flappybird` due to how app discovery works.

Reply:
ok Ty